x11-libs/libx11/libx11-1.6.5.recipe

```diff
@@ -148,7 +161,7 @@ INSTALL()
 	fixPkgconfig
 
 	if [ -n "$secondaryArchSuffix" ]; then
-		rm -rf $documentationDir
+		rm -rf $documentationDir/man
```

Comment:
You could as well use `manDir` instead

Reply:
I was only following some portions that already included it, in this case it would be removed because $documentationDir is deleted, and yes, $manDir should probably fit better here :)